pkg/apis/crd/v1beta1/types.go

```diff
@@ -1004,9 +1004,17 @@ type EgressSpec struct {
 	// Cannot be set with ExternalIPPool.
 	ExternalIPPools []string `json:"externalIPPools,omitempty"`
 	// Bandwidth specifies the rate limit of north-south egress traffic of this Egress.
-	Bandwidth *Bandwidth `json:"bandwidth,omitempty"`
+	Bandwidth     *Bandwidth        `json:"bandwidth,omitempty"`
+	FailurePolicy FailurePolicyType `json:"failurePolicy"`
 }
 
+type FailurePolicyType string
+
+const (
+	FailurePolicyDrop     FailurePolicyType = "Drop"
+	FailurePolicyNodeSNAT FailurePolicyType = "NodeSNAT"
```

Comment:
Not everyone could get the meanings of the two values. Please add comments to explain the behaviors.

Reply:
done